pkg/flowkit/services/accounts_test.go

```diff
@@ -217,4 +222,313 @@ func TestAccounts(t *testing.T) {
 		assert.NoError(t, err)
 	})
 
+	t.Run("Staking Info for Account", func(t *testing.T) {
+		_, s, gw := setup()
+
+		gw.ExecuteScript.Run(func(args mock.Arguments) {
+			assert.True(t, strings.Contains(string(args.Get(0).([]byte)), "import FlowIDTableStaking from 0x9eca2b38b18b5dfe"))
+			gw.ExecuteScript.Return(cadence.NewValue(nil))
+		})
+
+		val1, val2, err := s.Accounts.StakingInfo(flow.HexToAddress("df9c30eb2252f1fa"))
+		assert.NoError(t, err)
+		assert.NotNil(t, val1)
+		assert.NotNil(t, val2)
+	})
+
+}
+
+func setupIntegration() (*flowkit.State, *Services) {
+	readerWriter := tests.ReaderWriter()
+	state, err := flowkit.Init(readerWriter, crypto.ECDSA_P256, crypto.SHA3_256)
+	if err != nil {
+		panic(err)
+	}
+
+	acc, _ := state.EmulatorServiceAccount()
+	gw := gateway.NewEmulatorGateway(acc)
+	s := NewServices(gw, state, output.NewStdoutLogger(output.NoneLog))
+
+	return state, s
+}
+
+func TestAccountsCreate_Integration(t *testing.T) {
+	type accountsIn struct {
+		account  *flowkit.Account
+		pubKeys  []crypto.PublicKey
+		weights  []int
+		sigAlgo  crypto.SignatureAlgorithm
+		hashAlgo crypto.HashAlgorithm
+		args     []string
+	}
+
+	type accountsOut struct {
+		address  string
+		code     map[string][]byte
+		balance  uint64
+		sigAlgo  crypto.SignatureAlgorithm
+		hashAlgo crypto.HashAlgorithm
+		pubKeys  []crypto.PublicKey
+		weights  []int
+	}
+
+	t.Run("Create", func(t *testing.T) {
+		state, s := setupIntegration()
+		srvAcc, _ := state.EmulatorServiceAccount()
+
+		accIn := []accountsIn{{
+			account:  srvAcc,
+			sigAlgo:  crypto.ECDSA_P256,
+			hashAlgo: crypto.SHA3_256,
+			args:     nil,
+			pubKeys: []crypto.PublicKey{
+				tests.PubKeys()[0],
+			},
+			weights: []int{10000},
```

Comment:
```suggestion
			weights: []int{1000},
```

Should this be 1000?